session4-6/personalwebsite/src/pages/home.hbs

```diff
@@ -0,0 +1,25 @@
+---
+title: Home
```

Comment:
Use partial syntax to pass JSON data. Create separate JSON models and process them in server.js